src/extension/wps/src/test/java/au/org/emii/ncdfgenerator/GenerationIT.java

```diff
@@ -179,54 +183,55 @@ private Connection getConn() throws Exception {
         Properties props = new Properties();
         props.setProperty("user", env.get("POSTGRES_USER"));
         props.setProperty("password", env.get("POSTGRES_PASS"));
-        props.setProperty("ssl", "true");
-        props.setProperty("sslfactory", "org.postgresql.ssl.NonValidatingFactory");
+        props.setProperty("database", env.get("POSTGRES_DB"));
+/*        props.setProperty("ssl", "true");
+        props.setProperty("sslfactory", "org.postgresql.ssl.NonValidatingFactory");*/
```

Comment:
I reckon we should be setting it up to run one way.  Any reason why we can't connect to the db the same way every time - probably using ssl as thats what we do elsewhere.  No need to cater for manual changes here.